pyproject.toml

```diff
@@ -34,14 +34,13 @@ dependencies = [
     "scikit-learn",
     "scipy",
     "torch >= 2.1",
-    "torch_scatter",
     "astartes[molecules]",
 ]
 
 [project.optional-dependencies]
 dev = ["black", "bumpversion", "flake8", "pytest", "pytest-cov"]
 docs = ["nbsphinx", "sphinx", "sphinx-argparse", "sphinx-autobuild", "sphinx-autoapi", "sphinxcontrib-bibtex", "sphinx-book-theme","nbsphinx-link","ipykernel"]
-test = ["parameterized > 0.8", "pytest >= 6.2", "pytest-cov"]
```

Comment:
What did we use this for and why do we remove it?

Reply:
`parameterized` is used for running tests, but we don't use it anywhere (anymore, at least)